user_interaction/templates/user_interaction/account_pages/site_account_page.html

```diff
@@ -18,6 +28,16 @@ <h1>Account</h1>
         <small class="text-muted">This information can currently not be edited on your end. Please notify the <a href="mailto:uups@kotkt.nl">UUPS</a> if something needs to be changed.</small>
     </p>
     <p>
-        You can change your password <a href="{% url 'core:user_accounts/password_change' %}">here</a>
+        You can change your password <a href="{% url 'account:password_change' %}">here</a>
     </p>
+
+    <h3>Look and layout</h3>
+    <p>
+        Site theme: {{ user.preferences.layout__theme }}
```

Comment:
This shows the internal name of the theme instead of the human-readable variant. Fine enough for now.